src/components/Card.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const Card = (props) => {
+  return (
+    <div className="col-sm-12 col-md-4">
+      <div className="card text-center">
+        <img className="card-img-top" src={props.image} alt="Card image cap" />
+        <div className="card-body">
+          <p className="card-text">
+            <b>{props.name}</b>
```

Comment:
This is deprecated on HTML5 better use <strong> for bold text or change it in CSS